### PR TITLE
BAEL-7055: How can I capitalize the first letter of each word in a string

### PR DESCRIPTION
**Pull request by azhwani**
+ Create a new module for string operations: core-java-string-operations-7
+ Create methods to capitalize the first letter of each word in a given string
+ Create test case for each method

### CODE REVIEW

**Review by JavierPeris — APPROVED** [review]

File: core-java-modules/core-java-string-operations-7/src/main/java/com/baeldung/capitalizefirstcharactereachword/CapitalizeFirstCharacterEachWordUtils.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        }
+
+        return Arrays.stream(input.split("\\s+"))
+          .map(StringUtils::capitalize)
+          .collect(Collectors.joining(" "));
+    }
+
+    static String usingWordUtilsClass(String input) {
+        return WordUtils.capitalizeFully(input);
+    }
```
Comment (KevinGilmore) [inline core-java-modules/core-java-string-operations-7/src/main/java/com/baeldung/capitalizefirstcharactereachword/CapitalizeFirstCharacterEachWordUtils.java:43]: We can omit this and just test the WordUtils.capitalizeFully method directly.

File: core-java-modules/core-java-string-operations-7/src/test/java/com/baeldung/capitalizefirstcharactereachword/CapitalizeFirstCharacterEachWordUtilsUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        String input = "life is short the world is wide";
+
+        assertEquals("Life Is Short The World Is Wide", CapitalizeFirstCharacterEachWordUtils.usingStringUtilsClass(input));
+    }
+
+    @Test
+    void givenString_whenUsingWordUtilsClass_thenCapitalizeFirstCharacter() {
+        String input = "smile sunshine is good for your teeth";
+
+        assertEquals("Smile Sunshine Is Good For Your Teeth", CapitalizeFirstCharacterEachWordUtils.usingWordUtilsClass(input));
```
Comment (KevinGilmore) [inline core-java-modules/core-java-string-operations-7/src/test/java/com/baeldung/capitalizefirstcharactereachword/CapitalizeFirstCharacterEachWordUtilsUnitTest.java:34]: As mentioned above, let's just use WordUtils.capitalizeFully directly in this test.

**Review by KevinGilmore — COMMENTED** [review]
Check tab indentation in pom.xml and simplify WordUtils test case.

File: core-java-modules/core-java-string-operations-7/pom.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${apache.commons-lang.version}</version>
+        </dependency>
+        <dependency>
+		    <groupId>org.apache.commons</groupId>
+		    <artifactId>commons-text</artifactId>
+		    <version>${apache.commons-text.version}</version>
+		</dependency>
```
[the remark this answers is not in the document]
Comment (azhwani) [reply core-java-modules/core-java-string-operations-7/pom.xml]: done

File: core-java-modules/core-java-string-operations-7/pom.xml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                <configuration>
+                    <source>${maven.compiler.source}</source>
+                    <target>${maven.compiler.target}</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+		<maven.compiler.source>11</maven.compiler.source>
```
[the remark this answers is not in the document]
Comment (azhwani) [reply core-java-modules/core-java-string-operations-7/pom.xml]: done

File: core-java-modules/core-java-string-operations-7/src/main/java/com/baeldung/capitalizefirstcharactereachword/CapitalizeFirstCharacterEachWordUtils.java (hunk 1 above)
Comment (azhwani) [reply core-java-modules/core-java-string-operations-7/src/main/java/com/baeldung/capitalizefirstcharactereachword/CapitalizeFirstCharacterEachWordUtils.java]: done

**Pull request merged**